### PR TITLE
Setup and configure ISMRM conference pages, Aunt Minnie and Aunt Minnie Europe

### PR DESCRIPTION
**Pull request by jwade1327**
Screenshot 2024-04-23 at 8 22 22 AM

Screenshot 2024-04-22 at 11 24 11 AM
Screenshot 2024-04-22 at 11 30 07 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[Setup ISMRM on AM and AME](https://trello.com/c/dSvYAQUc/555-setup-ismrm-on-am-and-ame)

**Review by Shinsina — CHANGES_REQUESTED** [review]
It doesn't look like `/resources/conference/ismrm/2024` works on AuntMinnie Europe, it looks like it's supposed to given the changes in this PR.

image

File: sites/auntminnieeurope.com/config/page-details.js [hunk 1]
```diff
@@ -26,4 +31,21 @@ module.exports = {
...
+    },
+    primaryImage: {
+      src: 'https://img.auntminnie.com/files/base/smg/all/image/static/am/conference/ISMRM.png',
+      alt: 'ISMRM News Coverage',
+    },
+    teaserCTA: 'View Latest Coverage',
+  },
+  'resources/conference/ismrm/2024': {
+    name: '2024 International Society of Magnetic Resonance (ISMRM) News Coverage',
+    siblingRoutes: ismrmSiblingRoutes,
```
Comment (Shinsina) [inline sites/auntminnieeurope.com/config/page-details.js:49]: Do these also need to be corrected to `/resources/conferences`?

File: sites/auntminnieeurope.com/config/page-details.js [hunk 2]
```diff
@@ -2,11 +2,16 @@ const ecrSiblingRoutes = [
   { title: 2024, href: 'resources/conferences/ecr/2024', alt: '2024 European Congress of Radiology (ECR) News Coverage' },
 ];
 
+const ismrmSiblingRoutes = [
+  { title: 2024, href: 'resources/conference/ismrm/2024', alt: '2024 Radiological Society of North America (ISMRM) News Coverage' },
+];
+
```
Comment (Shinsina) [inline sites/auntminnieeurope.com/config/page-details.js:8]: Likewise here does this need to be changed to `resources/conferences/ismrm/2024`?

**Review by Shinsina — CHANGES_REQUESTED** [review]

File: sites/auntminnieeurope.com/config/page-details.js (hunk 2 above)
Comment (jwade1327) [reply sites/auntminnieeurope.com/config/page-details.js]: @Shinsina Apologies!!!

**Review by Shinsina — APPROVED** [review]

**Pull request merged**